src/Shared/src/PropertyHelper/PropertyHelper.cs

```diff
@@ -145,7 +144,7 @@ public static PropertyHelper[] GetProperties(TypeInfo typeInfo)
         /// </returns>
         public static PropertyHelper[] GetProperties(Type type)
         {
-            return GetProperties(type, CreateInstance, PropertiesCache);
+            return GetProperties(type, (p) => CreateInstance(p), PropertiesCache);
```

Comment:
Nit: just write `p => CreateInstance(p)` to be consistent with other places.

Look at the others too.

Reply:
done